auth0/structure_auth0_tenant.go

```diff
@@ -113,22 +115,24 @@ func expandTenantErrorPage(d ResourceData) *management.TenantErrorPage {
 }
 
 func expandTenantFlags(d ResourceData) *management.TenantFlags {
-	var flags management.TenantFlags
+	var flags *management.TenantFlags
```

Comment:
By changing this to a pointer and only setting the flags if we have the flags list set in the terraform config, we won't force the flags to be sent as an empty object any more to avoid the 400 bad requests error we get from the API in case we don't have any flags sent within. This truly makes the flags now optional as the docs indicate.